com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -544,5 +544,26 @@ public Vector2Int GetLastScaledSize()
         {
             return m_LastScaledSize;
         }
+
+        /// <summary>
+        /// Returns the resolved low res multiplier based on the low transparency threshold settings.
+        /// </summary>
+        /// <param name="targetLowRes"> the target low resolution.
+        ///     If by any chance thresholding is disabled or clamped, the exact same resolution is returned.
+        ///     This allows the caller to directly compare the float result safely with the floating point target resolution.
+        /// </param>
+        /// <returns>Returns the resolved low res multiplier based on the low transparency threshold settings.</returns>
+        public float GetLowResMultiplier(float targetLowRes)
```

Comment:
I actually wonder if we need this on the handler? Not all pipelines will have low res stuff? 

I guess as stop gap I would make the documentation more generic (e.g. "The pipeline can use this to drive the scale for low res transparency if available etc.")

Reply:
Updated doc comment